Makefile

```diff
@@ -36,22 +44,23 @@ export LUA_PATH
 all: build split-tests
 
 clean: clean-submodules
-	rm -rf $(BUILD_DIR)
+	rm -rf $(DEFN_DIR)
+	git clean -dfx -- tests/specs
 
 clean-submodules:
 	rm -rf $(DEPS_DIR)/k/make.timestamp $(DEPS_DIR)/pandoc-tangle/make.timestamp $(DEPS_DIR)/metropolis/*.sty \
 	       tests/ethereum-tests/make.timestamp tests/proofs/make.timestamp $(DEPS_DIR)/plugin/make.timestamp
 
 distclean: clean
-	cd $(K_SUBMODULE) && mvn clean -q
-	git submodule deinit --force -- ./
+	rm -rf $(BUILD_DIR) $(DEPS_DIR)
```

Comment:
Do you mean because we're removing the git submodules, and then re-initing them, or because it's weird to remove the deps dir in general? This is the simplest way I could think of to force a full clean rebuild of all our dependencies, but I wanted `make clean` to be more useful in general so it shouldn't remove the dependencies (it only removes the `DEFN_DIR`).

Reply:
Fixed.